### ISSUE TITLE
Decoder output

### ISSUE THREAD

**Issue by qq583** [JulesBelveze/time-series-autoencoder#17]
i can't understand that AttnDecoder return the 'self.fc_out(torch.cat((h_t[0], context.to(device)), dim=1))' . i can‘t find the correlation between the 'h_t' and 'context',plz tell me. Thank you!!!!!!!!!!!!!!!!
